action.yml

```diff
@@ -0,0 +1,11 @@
+# action.yml
+name: 'NLU Hyperopt'
+description: 'Find optimal hyperparameters for NLU model'
+outputs:
```

Comment:
we need to describe the inputs for the actions as well please!

Reply:
I had some inputs when I thought I could use `--build-args`, but then I removed them when I realized that didn't work. Env vars don't count as inputs do they?

Reply:
ah, now I see inputs become environmental variables!

Reply:
inputs added